.github/workflows/ci.yml

```diff
@@ -43,7 +44,7 @@ jobs:
         options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
```

Comment:
As I was trying to figure out why things failed, I figured I might as well align this to other yml files.